deploy/acm-policies/50-GENERATED-rosa-ingress-certificate-policies.Policy.yaml

```diff
@@ -24,69 +24,25 @@ spec:
                     - complianceType: mustonlyhave
                       metadataComplianceType: musthave
                       objectDefinition:
-                        apiVersion: policy.open-cluster-management.io/v1
-                        kind: Policy
+                        apiVersion: v1
+                        data:
+                            tls.crt: '{{hub fromSecret "openshift-acm-policies" .ManagedClusterName "tls.crt" hub}}'
+                            tls.key: '{{hub fromSecret "openshift-acm-policies" .ManagedClusterName "tls.key" hub}}'
+                        kind: Secret
                         metadata:
-                            annotations:
-                                policy.open-cluster-management.io/categories: CM Configuration Management
-                                policy.open-cluster-management.io/controls: CM-2 Baseline Configuration
-                                policy.open-cluster-management.io/standards: NIST SP 800-53
-                            name: rosa-ingress-certificate
-                            namespace: openshift-acm-policies
+                            name: ingress-certificate-secret
```

Comment:
Can we name the secret ` <cluster-name>-primary-cert-bundle-secret` to be consistent with rosa classic?

Reply:
name: '{{hub (printf "%s-primary-cert-bundle-secret" .ManagedClusterName) hub}}'

should work, testing now

Reply:
```
oc get secrets -n openshift-ingress
NAME                                                          TYPE                                  DATA   AGE
22kjvnk3l2r0jfrquk5faja581qha0en-primary-cert-bundle-secret   Opaque                                2      9m48s
```